notebook/services/kernels/kernelmanager.py

```diff
@@ -117,7 +123,7 @@ def _default_kernel_buffers(self):
         help="The last activity on any kernel, including shutting down a kernel")
 
     def __init__(self, **kwargs):
-        super(MappingKernelManager, self).__init__(**kwargs)
+        super().__init__(**kwargs)
         self.last_kernel_activity = utcnow()
```

Comment:
Should be removed?

Reply:
Haha I just added mentioned this too! :+1: